src/ORM/Behavior/CounterCacheBehavior.php

```diff
@@ -105,10 +105,15 @@ class CounterCacheBehavior extends Behavior
      *
      * @param \Cake\Event\Event $event The beforeSave event that was fired
      * @param \Cake\Datasource\EntityInterface $entity The entity that is going to be saved
+     * @param \ArrayObject $options The options for the query
      * @return void
      */
-    public function beforeSave(Event $event, EntityInterface $entity)
+    public function beforeSave(Event $event, EntityInterface $entity, $options)
     {
+        if (isset($options['ignoreCounterCache']) && $options['ignoreCounterCache'] === true) {
```

Comment:
Could this option be added to the API docs? Right now you have to source dive to find it.